static/app/utils/replays/hooks/useReplayList.tsx

```diff
@@ -44,8 +46,11 @@ function useReplayList({
     api.clear();
     setData(prev => ({
       ...prev,
-      isFetching: true,
+      isFetching: enabled ? true : false,
```

Comment:
```suggestion
      isFetching: Boolean(enabled),
```

Nit: This could be written as `Boolean(enabled)` to convert the `undefined` into `false`, instead of using the ternary. 

Using the ternary is a slight extra bit of reading/thinking to make sure that the order of true/false wasn't swapped. If swapping were intended then `!enabled` would be enough.